metadata/src/main/java/org/apache/kafka/controller/metrics/ControllerMetadataMetrics.java

```diff
@@ -39,10 +42,20 @@
  * {@link org.apache.kafka.controller.metrics.QuorumControllerMetrics}, not here.
  */
 public final class ControllerMetadataMetrics implements AutoCloseable {
+    public static final int BROKER_UNREGISTERED_STATE = -1;
```

Comment:
Can you please create an Enum for the broker registration states? Also be sure to add an interface annotation "stable"